core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.component.ts

```diff
@@ -260,7 +259,7 @@ export class QuestionSuggestionReviewModalComponent
     this.refreshActiveContributionState();
   }
 
-  invertMap(originalMap: unknown): unknown {
+  invertMap(originalMap: Object): Object {
```

Comment:
Can you use a more specific type?

Reply:
I will handle this while strict typing this file. This file is not yet fully strict typed.

Reply:
thanks @JeeveshGarg